src/test/ui/rfcs/rfc-2396-target_feature-11/fn-traits.stderr

```diff
@@ -0,0 +1,39 @@
+error[E0277]: expected a `std::ops::Fn<()>` closure, found `fn() {foo}`
+  --> $DIR/fn-traits.rs:21:10
+   |
+LL | fn call(f: impl Fn()) {
+   |                 ---- required by this bound in `call`
+...
+LL |     call(foo);
+   |          ^^^ expected an `Fn<()>` closure, found `fn() {foo}`
+   |
+   = help: the trait `std::ops::Fn<()>` is not implemented for `fn() {foo}`
+   = note: wrap the `fn() {foo}` in a closure with no arguments: `|| { /* code */ }
```

Comment:
Maybe add a note or change this note?

Reply:
FWIW, this *is* misleading -- but then I think I would like to make this suggestion actually work, that's what https://github.com/rust-lang/rust/issues/73631 is about.